src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -101,17 +103,11 @@ public sealed class CompletionItem : IComparable<CompletionItem>
         public CompletionItemRules Rules { get; }
 
         /// <summary>
-        /// Returns true if this item's text edit requires complex resolution that
-        /// may impact performance. For example, an edit may be complex if it needs
-        /// to format or type check the resulting code, or make complex non-local
-        /// changes to other parts of the file.
-        /// Complex resolution is used so we only do the minimum amount of work
-        /// needed to display completion items. It is performed only for the
-        /// committed item just prior to commit. Thus, it is ideal for any expensive
-        /// completion work that does not affect the display of the item in the
-        /// completion list, but is necessary for committing the item.
-        /// An example of an item type requiring complex resolution is C#/VB
-        /// override completion.
+        /// Returns true if this item's text edit requires complex resolution.
+        /// An edit is considered complex if the span of the change is different from
+        /// specified by <see cref="Span"/>.
+        /// 
+        /// Example of an item type requiring complex resolution is C#/VB override completion.
         /// </summary>
         public bool IsComplexTextEdit { get; }
```

Comment:
This was used as a hint for expensive operation before. Now it means the change resulted from committing the item is different from the default completion list span.